cms/urls.py

```diff
@@ -4,10 +4,9 @@
 
 from cms.apphook_pool import apphook_pool
 from cms.appresolver import get_app_patterns
+from cms.constants import SLUG_REGEXP
 from cms.views import details
 
-# This is a constant, really, but must live here due to import order
```

Comment:
@yakky can you, please check if this is really the issue or move this constant to some other place instead?